packages/apm/src/integrations/tracing.ts

```diff
@@ -350,6 +371,25 @@ export class Tracing implements Integration {
     });
   }
 
+  /**
+   * Uses logger.log to log things in the SDK or as breadcrumbs if defined in options
+   */
+  private static _log(...args: any[]): void {
+    if (Tracing.options.writeInternalDebugLogsAsBreadcrumbs) {
+      const _getCurrentHub = Tracing._getCurrentHub;
+      if (_getCurrentHub) {
+        _getCurrentHub().addBreadcrumb({
+          category: 'tracing',
+          level: Severity.Debug,
+          message: args.join(' '),
```

Comment:
We have a `safeJoin` for things like this - https://github.com/getsentry/sentry-javascript/blob/master/packages/integrations/src/captureconsole.ts#L57